apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -0,0 +1,118 @@
+
+export default function CurriculumQuickAssign() {
+  const [courseOfferings, setCourseOfferings] = useState(null);
+  const [decideLater, setDecideLater] = useState(false);
+  const [marketingAudience, setMarketingAudience] = useState(null);
+
+  // Retrieve course offerings on mount and convert to JSON
+  useEffect(() => {
+    fetch('/course_offerings/quick_assign_course_offerings')
+      .then(response => response.json())
+      .then(data => setCourseOfferings(data));
+  }, []);
+
+  /*
+  When toggling 'decide later', clear out marketing audience or assign one to make
+  the table appear again automatically.
+  */
+  const toggleDecideLater = () => {
+    setDecideLater(!decideLater);
+    if (marketingAudience !== '') {
+      setMarketingAudience('');
+    } else {
+      setMarketingAudience('elementary');
+    }
+  };
+
+  // When selecting a marketing audience, ensure 'decide later' is unchecked
+  const updateMarketingAudience = marketingAudience => {
+    setMarketingAudience(marketingAudience);
+    setDecideLater(false);
+  };
+
+  return (
+    <div>
+      <h2>{i18n.assignACurriculum()}</h2>
+      <h5>{i18n.useDropdownMessage()}</h5>
+      <div style={styles.buttonRow} className="quick-assign-top-row">
+        <div style={styles.buttonsInRow}>
+          <Button
+            id={'uitest-elementary-button'}
+            style={styles.buttonStyle}
+            text={i18n.courseBlocksGradeBandsElementary()}
+            size={Button.ButtonSize.large}
+            icon={
+              marketingAudience === 'elementary' ? 'caret-up' : 'caret-down'
+            }
+            onClick={() => updateMarketingAudience('elementary')}
+          />
+          <Button
+            id={'uitest-middle-button'}
+            style={styles.buttonStyle}
+            text={i18n.courseBlocksGradeBandsMiddle()}
+            size={Button.ButtonSize.large}
+            icon={marketingAudience === 'middle' ? 'caret-up' : 'caret-down'}
+            onClick={() => updateMarketingAudience('middle')}
+          />
+          <Button
+            id={'uitest-high-button'}
+            style={styles.buttonStyle}
+            text={i18n.courseBlocksGradeBandsHigh()}
+            size={Button.ButtonSize.large}
+            icon={marketingAudience === 'high' ? 'caret-up' : 'caret-down'}
+            onClick={() => updateMarketingAudience('high')}
+          />
+          <Button
+            id={'uitest-hoc-button'}
+            style={styles.buttonStyle}
+            text={i18n.courseOfferingHOC()}
+            size={Button.ButtonSize.large}
+            icon={marketingAudience === 'hoc' ? 'caret-up' : 'caret-down'}
+            onClick={() => updateMarketingAudience('hoc')}
+          />
+          <input
+            checked={decideLater}
+            style={styles.inputStyle}
+            type="checkbox"
+            id={'decide-later'}
+            onChange={toggleDecideLater}
+          />
+          <h5>{i18n.decideLater()}</h5>
```

Comment:
Should this be associated as a label or something with the input above?

Reply:
Good call- I fixed this:)